.infrastructure/55_cognito.tf

```diff
@@ -22,6 +23,44 @@ module "cognito_custom_message_function" {
   }
 }
 
+module "cognito_post_confirmation_function" {
+  source = "terraform-aws-modules/lambda/aws"
+
+  function_name = "cognito_post_confirmation"
+  description   = "The Lambda function executed after post confirmation of email address"
+  handler       = "main.sensEmailHandler"
+  runtime       = "nodejs18.x"
+
+  create_package                          = false
+  local_existing_package                  = "../apps/cognito-functions/out/cognito-functions.zip"
```

Comment:
We can use a locals, [the value is duplicated](https://github.com/pagopa/developer-portal/pull/361/files#diff-da868fc445e8049696f8dbb61dccdaf98813472d096da8dac9292b364398515bL10)

Reply:
I made some changes